vendor/bytebuddy-1_10_8/build.gradle

```
@@ -28,6 +28,9 @@ vendorJava(
   relocations: [
     "net.bytebuddy": "org.apache.beam.vendor.bytebuddy.v1_10_8.net.bytebuddy"
   ],
+  exclusions: [
+    "META-INF/versions/9/module-info.class"
```

We should exclude it regardless where it appears. This also matches how its being excluded in vendored gRPC.
```suggestion
    "module-info.class"
```

---

will do, I tried the version you proposed and it did not match it, but `**/module-info.class` does.